package.json

```diff
@@ -3,12 +3,14 @@
   "version": "0.0.0",
   "description": "electron-react-redux-boilerplate",
   "main": "init.js",
-  "author": "Jordan Schroter",
+  "author": {
+    "name": "Jordan Schroter",
+    "email": "email@author.com"
```

Comment:
I think this will be changed anyway since it's boilerplate project. We can leave it as is.